### PR TITLE
This fixes handling of Suppliers 

### LINKED ISSUE #70: As a volunteer I would like to know places to purchase watter bottles
* Size
* Price
* Contact
* Location

### PR DESCRIPTION
**Pull request by sanathks**
This PR will resolve #70, As a volunteer, I would like to place to purchase goods

### CODE REVIEW

**Comment by sanathks** [comment]
@gayanhewa @thinkholic @chamilad @danishka, Please review this

File: config/app.php [hunk 1]
```diff
@@ -167,7 +167,7 @@
          * Package Service Providers...
          */
         Laravel\Tinker\TinkerServiceProvider::class,
-        AlbertCht\InvisibleReCaptcha\InvisibleReCaptchaServiceProvider::class,
+        //AlbertCht\InvisibleReCaptcha\InvisibleReCaptchaServiceProvider::class,
```
Comment (gayanhewa) [inline config/app.php:170]: Are we removing the invisible ReCaptcha?

**Review by gayanhewa — APPROVED** [review]
LGTM 👍, apart from the question on removing the recaptcha.

File: config/app.php (hunk 1 above)
Comment (sanathks) [reply config/app.php:170]: Sorry, It was a mistake. Thanks for pointing

**Comment by kasunben** [comment]
@gayanhewa @naveendw did you guys test this on local? Let me know, then I can take this and merge.

@sanathks any DB changes needed?

**Comment by sanathks** [comment]
@thinkholic Yes, it is. You have to run migrations and the seeder

**Comment by kasunben** [comment]
@sanathks Some conflicts there. Can you resolve so we can take this. :)

**Comment by sanathks** [comment]
@thinkholic I rebased my branch from upstream. Now conflicts are resolved. hope it's works

**Comment by danishka** [comment]
@sanathks would you mind to fix these conflicts and let the dev room know.
As someone can locally test it.

**Comment by sanathks** [comment]
I'm closing this PR since it's too old now ;)

**Pull request closed without merging**